### DISCUSSION TITLE
Profile pic not changing?

### DISCUSSION THREAD

**Discussion by MidhiaQueiroz** [community/community#69303] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I  want to upload photo to github profile, but it has error like this.
"Something went really wrong and we can’t process that picture."
I have already changed image size  but result is the same.

**Comment by mehulrajdev** [comment, marked as answer]
Check the file size: Github has a file size limit of 10 MB for individual files. If the picture you are trying to upload is larger than this, you will get an error message. Try reducing the size of the picture by compressing it or resizing it.

Try a different browser: If you are using a browser that is not supported by Github, you may encounter errors when uploading files. Try using a different browser such as Google Chrome or Firefox.

**Reply by igweze** [reply]
Its happening to me and I'm not receiving any error message, it will just be loading forever

**Reply by redhero2109** [reply]
mine works it just doesnt change, it changes on my page but not github itself

edit: im stupid it did work

**Comment by Akbaroke** [comment]
here are a few things you can try to fix the "Something went really wrong and we can't process that picture" error when uploading a photo to your GitHub profile:

Try a different browser. GitHub supports Chrome, Firefox, Edge, and Safari. If you're using a different browser, try switching to one of these supported browsers.
Try uploading the photo from a different device. If you're still getting the error, try uploading the photo from a different device, such as a different computer or smartphone.
Check the file format. GitHub only supports JPEG, PNG, and GIF image formats. If your photo is in a different format, try converting it to one of these supported formats.
Check the file size. GitHub has a file size limit of 10 MB for profile photos. If your photo is larger than 10 MB, try compressing it or resizing it.
Try uploading the photo again later. It's possible that there is a temporary problem with GitHub's servers. Try uploading the photo again later and see if the error goes away.
If you've tried all of the above and you're still getting the error, you can contact GitHub support for help.
I hope this helps!

**Comment by MidhiaQueiroz** [comment]
ok

**Reply by SheikhElMoctarG** [reply]
I tried all these solutions, but it's not working

**Reply by rockythef0x** [reply]
Same here

**Reply by AhamedPgx** [reply]
same here 
I am also no able to upload my proflle pic

**Reply by razaashai95** [reply]
same

**Reply by blmatthews** [reply]
Same here. My picture is a 74k png, and shows up as my profile picture on github.com/settings/profile, but nowhere my avatar is shown. I'm using the latest Chrome.

**Reply by mbaldris-pcd** [reply]
Same here

**Reply by blmatthews** [reply]
> Same here. My picture is a 74k png, and shows up as my profile picture on github.com/settings/profile, but nowhere my avatar is shown. I'm using the latest Chrome.

Update! After a couple of days mine magically appeared. It now shows up everywhere I could find that shows my avatar on github.

**Comment by mehmoodulhaq570** [comment]
It can happen because of your image size or image format issue; try to change the format of the picture and upload it again.

**Reply by dragonsdenexotech** [reply]
png wasnt working. jpg worked for me.

**Comment by RabbitDev810** [comment]
You can upload the photo. But your photo's size must less 10MB.

**Comment by dike-okpan** [comment]
I have been having issues with image upload on my profile and also to my repo. My image size is okay. Please how do I fix this?

**Comment by JL-ghcoder** [comment]
This should caused by ur image size i guess

**Comment by priyanshirabadiya** [comment]
I can upload picture in profile but can not set or customize the image - Crop your new profile picture  This is only loading not give response. Is it happens with anyone else?

**Comment by Harshil5786** [comment]
clear cache (press ctrl + F5)

**Comment by dharanidh-02** [comment]
crop the profile photo properly reduce the size so it will fit

**Comment by Antushil11** [comment]
Use a small square JPG/PNG (e.g., 500×500, <1MB) and try another browser or clear cache
